parsers/nmparser/parse_lst_file.go

```diff
@@ -46,7 +48,13 @@ func ParseLstEstimationFile(lines []string) LstData {
 		case strings.Contains(line, "0DEFAULT SIGMA BOUNDARY"):
 			endParameterStucturesIndex = i
 		case strings.Contains(line, "#OBJV"):
-			ofvIndex = i
+			parseOFV(line, ofvDetails)
```

Comment:
could we actually change this to not take a pointer receiver, and instead use a struct directly and just return a modified version, to keep with the functional programming style of most of the parsing logic we have.

something like:

```
ofvDetails = parseOFV(line, ofvDetails)
```